gamemodes/terrortown/gamemode/roles/guesser/cl_guesser.lua

```diff
@@ -0,0 +1,365 @@
+local hook = hook
+local string = string
+
+local AddHook = hook.Add
+local StringUpper = string.upper
+
+-------------
+-- CONVARS --
+-------------
+
+local guesser_show_team_threshold = GetConVar("ttt_guesser_show_team_threshold")
+local guesser_show_role_threshold = GetConVar("ttt_guesser_show_role_threshold")
+local guesser_can_guess_detectives = GetConVar("ttt_guesser_can_guess_detectives")
+local guesser_unguessable_roles = GetConVar("ttt_guesser_unguessable_roles")
+
+------------------
+-- TRANSLATIONS --
+------------------
+
+AddHook("Initialize", "Guesser_Translations_Initialize", function()
+    -- Weapons
+    LANG.AddToLanguage("english", "guessingdevice_help_pri", "Press {primaryfire} to guess a player.")
+    LANG.AddToLanguage("english", "guessingdevice_help_sec", "Press {secondaryfire} to select a role.")
+    LANG.AddToLanguage("english", "guessingdevice_title", "Role Guesser Selection")
+
+    -- HUD
+    LANG.AddToLanguage("english", "guesser_selection", "Role Selected: ")
+
+    -- Target ID
+    LANG.AddToLanguage("english", "guesser_unguessable", "UNGUESSABLE")
+
+    -- Scoring
+    LANG.AddToLanguage("english", "score_guesser_guessed_by", "Guessed by")
+
+    -- Events
+    LANG.AddToLanguage("english", "ev_guesser_correct", "{guesser} correctly guessed {victim}'s role")
+
+    -- Events
+    LANG.AddToLanguage("english", "ev_guesser_incorrect", "{guesser} incorrectly guessed {victim}'s role")
+
+    -- Popup
+    LANG.AddToLanguage("english", "info_popup_guesser", [[You are {role}! {traitors} think you are {ajester} and you deal no
+    damage. However, you can use your role guesser to try and guess a player's role. Guess
```

Comment:
The indentation at the beginning of these lines are included in the string. Left-justify to fix
```suggestion
damage. However, you can use your role guesser to try and guess a player's role. Guess
```